.github/workflows/End2EndTest.yml

```diff
@@ -40,22 +40,24 @@ jobs:
       - name: Code Coverage
         uses: codecov/codecov-action@v1
   build-windows:
-    name: Build & Test - Windows, JDK ${{ matrix.java }}, Cloud AWS
+    name: Build & Test - Windows, JDK ${{ matrix.java }}, Cloud ${{ matrix.snowflake_cloud }}
     runs-on: windows-2022
     strategy:
       matrix:
         java: [ 8 ]
+        snowflake_cloud: [ 'AWS', 'AZURE', 'GCP' ]
     steps:
       - name: Checkout Code
         uses: actions/checkout@v2
       - name: Install Java ${{ matrix.java }}
         uses: actions/setup-java@v1
         with:
           java-version: ${{ matrix.java }}
-      - name: Decrypt profile.json (Windows)
+      - name: Decrypt profile.json for Cloud ${{ matrix.snowflake_cloud }} on Windows
         env:
           DECRYPTION_PASSPHRASE: ${{ secrets.PROFILE_JSON_DECRYPT_PASSPHRASE }}
-        run: gpg --quiet --batch --yes --decrypt --passphrase="${env:DECRYPTION_PASSPHRASE}" --output profile.json ./profile.json.gpg
+        run: |
+          ./scripts/decrypt_secret.sh ${{ matrix.snowflake_cloud }}
```

Comment:
This sh script will not work on windows.

Reply:
Oh I see. I now understand why you have gpg command here 👍

Reply:
I added a powershell script which does the same thing as sh script. Thanks Lukas for pointing that out.

Reply:
It's failing `testInsertRowThrottling`, so I guess I will wait for your other PR to merge.